tests/calculate.sh

```diff
@@ -1,11 +1,9 @@
 #!/bin/bash
-set -o nounset
```

Comment:
these set lines were removed, i think they are best practices to keep in the calculate.sh script or possibly need to be there for jenkins?

Reply:
I saw once jenkins report one of the error guarding setting (-o pipefail) invalid, feel strange though, I can add back to test again.